### ISSUE TITLE
Resident Evil 2 remake / Resident Evil 3 remake unable to launch game

### ISSUE THREAD

**Issue by whchan05** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#649]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Resident Evil 2 remake / Resident Evil 3 remake

### Processor / Processor Number [Required]

5700X

### Graphic Card [Required]

A770 LE

### GPU Driver Version [Required]

31.0.101.5081

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [ ] Not applicable

### Windows Build Number [Required]

- [ ] Windows 11 23H2
- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[output 19 Dec 23.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13742189/output.19.Dec.23.txt)

### Description and steps to reproduce [Required]

Basically title. I have seen other people playing these normally but I just cannot launch them. Happens even if I tried reinstalling Windows 3 months ago. Probably AMD CPU problem? I am not sure.

**Comment by Felipe-Intel** [comment]
Hello @whchan05.
We tried these games in a system with AMD processor paired with an A750. We were able to launch the game. So, since this is a crash, do you have any log or crash dump? Please check [this guide](https://learn.microsoft.com/en-us/troubleshoot/windows-client/performance/generate-a-kernel-or-complete-crash-dump) from Microsoft on how to enable crash dumps. Also check the folder "C:\Users\%USERNAME%\AppData\Local\..." for other dumps or logs. Additionally, sometimes there are logs and/or dump files in the steam folder (look for the specific game). And finally, share the [windows event log](https://learn.microsoft.com/en-us/shows/inside/event-viewer). You can export the logs for "Windows Logs/Application". 
With this we can try to determine what is happening in your system. 

In the meantime you can try the following. I guess you already did this but it is worth to mention, specially if this happens with other programs and/or games: 

1. Check all the connections in your motherboard. Re-seat GPU, RAM.
2. Verify PSU is in good shape.
3. Update BIOS, OS (make sure you are using a production version, not a beta or insiders version) and all drivers.
4. [verify windows files](https://support.microsoft.com/en-gb/topic/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system-files-79aa86cb-ca52-166a-92a3-966e85d4094e)
5. verify files or reinstall the game (try a different HDD/SSD)

You can also check in other forums and communities if anyone is having this problem and what they did about it.

**Comment by Felipe-Intel** [comment]
@Arturo-Intel @IGCIT above comment was posted last friday, Jan 5th. It is your call on closing this ticket. Thanks!
